packages/panamax/build-mirror.py

```
@@ -0,0 +1,352 @@
+                    kept.append(line)
+                    continue
+                if vers in available:
+                    kept.append(line)
+                else:
+                    removed += 1
+
+            if removed == 0:
+                total_kept += len(kept)
+                continue
+
+            has_content = any(l.strip() for l in kept)
```

Ruff (0.15.12)

[error] Ambiguous variable name: `l`

(E741)